front/src/style/globals.css

```diff
@@ -10,7 +10,6 @@
 
 @media (prefers-color-scheme: dark) {
     :root {
-        --foreground-rgb: 255, 255, 255;
```

Comment:
Changing the foreground colour to modify the text colour is not very intuitive... Could you add a screenshot to the PR?

Reply:
Yeah so the foreground-rgb is defined here but used later in the `body`, the `color` property there refers to the colour of the text's body. By removing the `--foreground-rgb: 255, 255, 255` (which is white), in the definition of dark mode colours, it defaults back to the `root` definition of `foreground-rgb` which is black!
<img width="1356" alt="Screenshot 2023-10-05 at 12 27 47" src="https://github.com/Theodo-UK/OmniLog/assets/47194161/c0d57ad4-5b01-4e24-893d-21aef050a7d3">